### PR TITLE
Antorus/Argus: Add messages for "Withering Roots"

### PR DESCRIPTION
**Pull request by Zireael-N**

### CODE REVIEW

File: Antorus/Argus.lua [hunk 1]
```diff
@@ -757,6 +761,53 @@ end
...
+			self:Message(args.spellId, "Attention", "Alert", L.stacks(1, args.spellName))
+		else -- for others: just warn that the tree is now withering
+			self:Message("stages", "Positive", "Long", args.spellId)
+		end
+	end
+
+	-- helper function with access to a local variable to avoid
+	-- passing by value and rescheduling the timer
+	local function announce(self, spellId, spellName)
+		self:Message(spellId, "Attention", "Alert", L.stacks(stacks, spellName))
```
Comment (elvador) [inline Antorus/Argus.lua]: Missing `:format`

Comment (Zireael-N) [reply Antorus/Argus.lua]: That's one of my favourite mistakes.

Comment (elvador) [reply Antorus/Argus.lua]: Been there, done that ;)

File: Antorus/Argus.lua [hunk 2]
```diff
@@ -757,6 +761,53 @@ end
...
+		scheduled = nil
+	end
+
+	function mod:WitheringRootsStacks(args)
+		if args.amount - lastAnnouncedStacks > 2 then -- normally it'll be 1, 4, 7...
+			local t = GetTime()
+			if t - prev > 1 then -- on Mythic players usually revive themselves simultaneously, we don't want to show multiple messages
+				prev = t
+				lastAnnouncedStacks = args.amount
+				self:Message(args.spellId, "Attention", "Alert", L.stacks:format(args.amount, args.spellName))
```
Comment (elvador) [inline Antorus/Argus.lua]: Is there a reason you don't call `announce()` here?
If you want to be really safe you could add a `if scheduled then self:CancelTimer(scheduled) end` as well.

Comment (Zireael-N) [reply Antorus/Argus.lua]: Nope, there wasn't a good reason. :s

**Pull request merged**